docs/tutorials/using-data-sources.md

```diff
@@ -9,28 +9,92 @@ import Tabs from '@theme/Tabs';
+            "trigger": {
+                "timeTrigger": {
+                    "every": 30
+                }
+            },
+            "filters": [{
+                    "key": {
+                        "name": "btc.price",
+                        "type": "TYPE_INTEGER",
+                        "numberDecimalPlaces": 8
+                    },
+                    "conditions": [
+                        {
+                            "operator": "OPERATOR_GREATER_THAN_OR_EQUAL",
+                            "value": "0"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+This will cause the network validators to read the result from the specified smart contract and submit the result to Vega. When the data is verified by enough validators, this price is accepted on to the network. This is unlike the other two Oracle types ([Open Oracle signed messages](#open-oracle-signed-messages) & [JSON signed message data](#json-signed-message-data)),which both rely on a third party submitting data to the network.
+
+### ABI
+The `address` field in the specification tells the spec above which address to interact with on Ethereum. The `abi` ([Application Binary Interface](https://docs.soliditylang.org/en/develop/abi-spec.html#json) & `method` field on the spec above tells the settlement spec **how** to interact with it. Ethereum Oracle settlement specifications use the JSON ABI of the smart contract to describe the method on the contract that will be called to fetch the data. The ABI will contain the function name, details of any paramters required, and the format of the response. 
+
+For example, the [Chainlink BTC/USD oracle](https://data.chain.link/ethereum/mainnet/crypto-usd/btc-usd) has its JSON ABI [published on Etherscan](https://etherscan.io/address/0xf4030086522a5beea4988f8ca5b36dbc97bee88c#code). When defining the data source spec, you can populate the `abi` field with the full ABI, and then set the `method` to `latestAnswer`.
+
+:::note Shrinking the ABI
+When populating the `abi` field on your data source spec, you can remove the methods and other fields that are not required by the oracle. We've done that in the sample data source above - only the `latestAnswer` method and its inputs and outputs are in the `abi` field.
 :::
 
-## Who can submit data
-Any Vega keypair can submit data. In the configuration for a market, a data source specification field dictates which data feeds it is interested in. In effect, it works as a filter. This specification means that the creator of an instrument for a market will choose in advance a price source, and which data fields the market requires to settle and terminate.
+### Time trigger
+With Ethereum data source specs the validators will read the specified smart contract and method detailed in the ABI. The `trigger` instructs the validators when to do this. In the smple above, it will be called every 30 seconds.
```

Comment:
```suggestion
With Ethereum data source specs the validators will read the specified smart contract and method detailed in the ABI. The `trigger` instructs the validators when to do this. In the sample above, it will be called every 30 seconds.
```